monitor/docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       - 9090:9090
 
   grafana:
-    image: grafana/grafana:6.7.5
+    image: grafana/grafana:7.5.2
```

Comment:
Can this break some of our panels ?

Reply:
I didn't notice anything, but possibly. Keep in mind though that we run 7.x in prod and in our zeebe-io cluster, so it should work fine.